src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -2071,6 +2071,7 @@ public void SourceItemMissing()
 #else
         [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/297")]
 #endif
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Yeah, the state file exists only when we have the full-fledged resource reader, meaning we have links to remote files to follow. I think we're as good as we can be for now; in a glorious future where our .NET Core GenerateResource task can do all of the things its Full flavor can, we should enable these tests everywhere.

The Mono implications aren't clear to me, but dropping down to the .NET Core behavior seems reasonable.